workspaces/github-pull-requests-board/plugins/github-pull-requests-board/src/plugin.ts

```diff
@@ -52,3 +54,17 @@ export const EntityTeamPullRequestsContent =
       mountPoint: rootRouteRef,
     }),
   );
+
+/** @public */
+export const HomePageTeamPullRequestsCard = homePlugin.provide(
```

Comment:
```suggestion
export const HomePageTeamPullRequestsCard = githubPullRequestsBoardPlugin.provide(
```

You don't set this on the `homePlugin` but for the current plugin. Then you can remove line 23 above and the reference to this package in the `package.json`